src/style.css

```
@@ -1,3 +1,89 @@
+:root {
+  --main-bg-color: #fff;
+  --main-active-color: #00acdc;
+  --main-text-color: #000;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  width: 100%;
+  font-size: 12px;
+  font-weight: 500;
```

Move these styles to the html selector
```suggestion
  font-family: Roboto, sans-serif;
  font-size: 12px;
  font-weight: 500;
```